Looport/Millennium-Falcon
# Create room message

# Description


This PR has a couple of purposes. 
One is to connect passport with telegraph service to receive user creation.
Second, to create room and messages that are sent via SSE. SEE implemented on BE and will be implemented on FE on the next task.

## What Changed


— Added NATS connection between passport and telegraph.
— Added room creation on “Start Call” click
— Added messaging via SEE (BE)
— Added message creation
— Added clean mocks for mocked services

## How To Test


— Login
— Click on “Start Call”
— You should be redirected to the room page
— Send message

# Checklist
- [x] I followed all possible cases of the implemented _user flow_
